src/components/AppLoader/AppLoader.component.js

```diff
@@ -43,6 +44,7 @@ export const AppLoader = (props: Props) => {
 
     const load = useCallback(async () => {
         try {
+            initFeatureAvailability(serverVersion);
```

Comment:
Hey, 
What is the use-case for which this initialization is needed in the `AppLoader`? You can find the `minorServerVersion` inside of the metadata factories. In the epics, the `serverVersion` is available in the redux store and within components by using the `useConfig()` app-runtime hook. 
Thank you!

Reply:
Thanks for bringing this up! :slightly_smiling_face: It turned out that all the places where I needed to check the server version was neither components nor hooks, but regular javascript functions. I couldn't see a convenient way to do this with the existing tools, so I made a new one. There isn't anything clever about how I did it, and I'm open to moving things around a little if it would make it feel more integrated.